examples/basic-mongo/package.json

```diff
@@ -9,7 +9,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@digipolis/auth": "^1.1.0",
+    "@digipolis/auth": "git+https://github.com/digipolisantwerp/auth_module_nodejs.git",
```

Comment:
Waarom is dit niet de gepubliceerde versie?

Reply:
omdat deze versie nog niet gepublished is.